Simplenote/Classes/ColorStudio.swift

```diff
@@ -0,0 +1,139 @@
+import Foundation
+
+
+// MARK: - Color Studio Constants
+//         Ref. https://github.com/Automattic/color-studio/blob/master/dist/colors.json
+//
+enum ColorStudio: String {
+    case black      = "000"
+    case blue0      = "eff3f7"
+    case blue5      = "c6def6"
+    case blue10     = "9ac6f0"
+    case blue20     = "72aee6"
+    case blue30     = "5198d9"
+    case blue40     = "3582c4"
+    case blue50     = "2271b1"
+    case blue60     = "135e96"
+    case blue70     = "0a4b78"
+    case blue80     = "043959"
+    case blue90     = "01263a"
+    case blue100    = "00131c"
+    case celadon0   = "#e4f2ed"
+    case celadon5   = "#a7e8d4"
+    case celadon10  = "#63d6b6"
+    case celadon20  = "#2ebd99"
+    case celadon30  = "#09a884"
+    case celadon40  = "#009172"
+    case celadon50  = "#007e65"
+    case celadon60  = "#006753"
+    case celadon70  = "#005042"
+    case celadon80  = "#003b30"
+    case celadon90  = "#002721"
```

Comment:
I see you don't use the _#_  prefix above. What's the right format do you want to use?

Reply:
Normalizing, thank you sir!!